courses/free-courses-pt_BR.md

```diff
@@ -243,6 +243,7 @@
 ### Node.js
 
 * [Criando APIs com NodeJs](https://www.youtube.com/playlist?list=PLHlHvK2lnJndvvycjBqQAbgEDqXxKLoqn) - Balta.io (YouTube)
+* [Curso de Node](https://www.youtube.com/watch?v=XN705pQeoyU&list=PLx4x_zx8csUjFC41ev2qX5dnr-0ThpoXE) - Professor Bruno CFBCursos (YouTube)
```

Comment:
we don't include honorifics such as "Prof."

Reply:
@eshellman adjusted